pipeline_noted.ipynb

```diff
@@ -99,14 +122,37 @@
+    "* `chunks` is used to divide data into smaller chunks for parallel processing. Currently it is rarelly used and has little impact on the actual performance of the script. \n",
+    "<div class=\"alert alert-info\">\n",
+    "It is recommended to leave chunks to default. \n",
+    "</div>\n",
+    "\n",
+    "\n",
     "* `in_memory` specify whether to load the whole dataset into memory.\n",
-    "* `interactive` controls whether interactive plots should be shown. Not that plotting the results implies actual computation to be carried out, and thus could be very inefficient when the data are not in the memory. In real situation you would probably want `in_memory = True` as well if your data can fit in the memory.\n",
-    "* `output_size` controls the realtive size of all the plots on a scale of 0-100 percent, though it can goes beyond 100 without any problem. Adjust this to please your eye."
+    "* `interactive` controls whether interactive plots should be shown.  Not that plotting the results implies actual computation to be carried out, and thus could be very inefficient when the data are not in the memory. \n",
+    "<div class=\"alert alert-info\">\n",
+    "In real situation, when you want to visualize interactive plots while figuring out the best parameters, you would want in_memory=True and interactive=True as long as your data can fit in the memory. On the other hand, once you finalized your parameters and ready for batch processing, you would want both of them to be set as False.\n",
+    "</div>\n",
+    "\n",
+    "\n",
+    "* `output_size` controls the realtive size of all the plots on a scale of 0-100 percent, though it can goes beyond 100 without any problem.\n",
+    "<div class=\"alert alert-info\">\n",
+    " Adjust this to please your eye.\n",
+    "</div> \n",
+    "\n",
+    "\n",
+    "* `param_save_minian` specify the data saving property for all the saving part. `dpath` is where you want the data to be saved, we recommand use the same dpath as where you load the imaging data. `fname` is the name of your dataset. In `backend`, `'zarr'` is designed for parallel and out-of-core computation, but its support is experimental for now and does not support incremental writing (i.e. it will be a pain to update part of exsisting dataset). `meta_dict` is a `dictionary` that is used to construct meta data for the final labeled data structure. \n",
+    "<div class=\"alert alert-info\">\n",
+    "The defult meta_dict is assuming data is stored in some structured folders as shown below. We do <strong> recommand </strong> users to structure their data like this to make the following steps (cross-registration) easier. However, you can simply adjust the value of meta_dict in this parameter to fit your data!\n",
```

Comment:
cross-registration does not care about folder structure at all

Reply:
It does, on the level that you need to keep meta_dict consistent